### PR TITLE
docs: fix sp run book

### PR DESCRIPTION
**Pull request by sysvm**

### CODE REVIEW

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md
Comment (maxnodereal) [inline docs/guide/storage-provider/run-book/run-testnet-SP-node.md:1]: ChainAddress = ['${chain_address}']
 which address should be put here?

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 1]
```diff
@@ -120,16 +120,16 @@ MetadataEndpoint = 'metadata:9333'
...
 HTTPAddress = '0.0.0.0:9033'
 
 [P2P]
 # p2p node msg Secp256k1 encryption key, it is different from other SP's addresses
 # P2PPrivateKey and node_id is generated by ./gnfd-sp p2p.create.key -n 1
-P2PPrivateKey = '' # writing to environment variables is recommended
+P2PPrivateKey = '' # recommend writing this config to environment variables
 P2PAddress = '0.0.0.0:9933'
 P2PAntAddress = '${load_balance_doamin:port}'
 P2PBootstrap = ['node_id@load_balance_doamin:port'] # p2p node's bootstrap node, format: [node_id1@ip1:port1, node_id2@ip1:port2]
```
Comment (maxnodereal) [inline docs/guide/storage-provider/run-book/run-testnet-SP-node.md:135]: how can users know the bootstrap nodes URL? can we add a section about "connect to Greenfield testnet" above?

**Comment by maxnodereal** [comment]
We can add a section to test SP:

Test your integration with [Dcellar](dcellar.io)
1. Connect with your own wallet
2.  Bridge some BNB to Greenfield testnet
3. Click “Create a bucket” and choose your own SP
4. Upload some files to your bucket.

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md
Comment (sysvm) [reply docs/guide/storage-provider/run-book/run-testnet-SP-node.md:1]: I add this below.

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md (hunk 1 above)
Comment (sysvm) [reply docs/guide/storage-provider/run-book/run-testnet-SP-node.md:135]: This section is used to connect to Greenfield testnet.

**Comment by sysvm** [comment]
> We can add a section to test SP:
> 
> Test your integration with [Dcellar](dcellar.io)
> 
> 1. Connect with your own wallet
> 2. Bridge some BNB to Greenfield testnet
> 3. Click “Create a bucket” and choose your own SP
> 4. Upload some files to your bucket.

I am not responsibile for Dcellar

**Review by RenRick — APPROVED** [review]

**Pull request merged**